analytics/loggable_objects.go

```diff
@@ -0,0 +1,96 @@
+package analytics
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"net/http"
+)
+
+type RequestType string
+
+const (
+	COOKIE_SYNC RequestType = "cookie_sync"
+	AUCTION     RequestType = "auction"
+	SETUID      RequestType = "set_uid"
+	AMP         RequestType = "amp"
+)
+
+//Loggable object of a transaction at /openrtb2/auction endpoint
+type AuctionObject struct {
+	Type      RequestType
```

Comment:
Why do these `Type` fields exist?

Reply:
The output log file consists of all the requests and responses. Type is to understand to what endpoint the request was made.